src/components/pages/dashboard/dashboard.container.js

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { translate } from 'react-i18next';
+import { Dashboard } from './dashboard';
+
+export const DashboardContainer = translate()(Dashboard);
```

Comment:
Question: Is translate that gives us localization support?

Reply:
Yes, it is a higher order component (like redux connect) that passes the translation methods through the props. https://react.i18next.com/components/translate-hoc.html